src/style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.card__container {
+  width: 200px;
+  background: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  display: block;
+  margin: 30px 19px 45px;
+  height: 134px;
+  width: 160px;
+}
+
+.description__block {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400px;
+  font-size: 11px;
+  padding: 0 14px;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 15px;
+}
+
+.title {
+  margin: 0 auto 5px;
+  font-weight: 500;
+  line-height: 1.25em;
+  font-size: 12px;
+}
+
+.product__code {
+  font-weight: 400;
+  margin: 0 0 15px;
+}
+
+.feedback {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.stars__container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: -3px;
+}
+
+.stars__item {
+  width: 15px;
+  height: 15px;
+  margin-left: 4px;
+  display: flex;
+  align-items: center;
+}
+
+.stars__container :last-child {
+  margin-right: 0;
+}
```

Comment:
use your code from previous task, so when I change class in html - count of active stars also changes